README.md

```diff
@@ -143,6 +143,7 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "useGitClone": true          Use a shallow git clone instead of downloading the module
 "ignoreGitHead":             Ignore the gitHead field if it exists and fallback to using github tags
 "yarn":                      Install and test the project using yarn instead of npm
+"testTimeout":               Number of milliseconds before timeout
```

Comment:
@targos what about just `timeout` to match the CLI arg? It applies to both `install` and `test`

Reply:
I agree with that.

Reply:
done